blustl/encode.py

```diff
@@ -0,0 +1,82 @@
+from functools import singledispatch
+import operator as op
+
+import numpy as np
+from lenses import lens
+
+import stl
+import pyeda
+
+@singledispatch
+def encode_stl(stl):
```

Comment:
Lets assume the STL has already had the temporal operators erased.

The function to do that is `blustl.game.discretize_stl`, but for now just take it as a precondition.